scrapper/idownloadcoupon.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import time
+import fetch
+import util
+
+
+async def get(session, url="https://idownloadcoupon.com/product-category/100off/"):
+    links_udemy = set()
+    start = time.time()
+
+    cont = await fetch.get_all(session, url)
+    if cont:
+        for html in cont:  # type: ignore
```

Comment:
Was this copy/pasted online? You aren't using `mypy`, so I don't see the reason for the type ignore comment. Safe to remove